dlna/dms/dms.go

```diff
@@ -267,6 +267,8 @@ type Server struct {
 	IgnoreHidden bool
 	// Ingnore unreadable files and directories
 	IgnoreUnreadable bool
+	// Ingnore comma separated list of directories
+	IgnorePaths string
```

Comment:
Could you make this one a `[]string`, and do the splitting when the flag is parsed instead of each time?